NEWS.md

```diff
@@ -3,6 +3,112 @@
 PinCFlow.jl follows the interpretation of
 [semantic versioning (semver)](https://julialang.github.io/Pkg.jl/dev/compatibility/#Version-specifier-format-1) used in the Julia ecosystem. Notable changes will be documented in this file for human readability.
 
-## Initial release v0.1
+## Release 0.1.0
 
   - First public release of PinCFlow.jl.
+
+## Release 1.0.0
```

Comment:
I would change the order so that the newest release notes are at the top of the list. This makes it easier for people to see which recent changes have been introduced.